content/articles/data-visualization-in-react-using-apexcharts/index.md

```diff
@@ -0,0 +1,260 @@
+```bash
+npx create-react-app charts-app
+```
+Once the process is done, run the following command to get into the charts-app folder.
+```bash
+cd charts-app
+```
+
+Next, install the ApexCharts.js library using the following command in your terminal.
+```bash
+npm install react-apexcharts apexcharts
+```
+
+In this article, we will use a JSON server to create a fake REST API that we will consume in our application. However, if you have an existing API endpoint, this won't be necessary and you can skip to the next step. For the readers who want to create a dummy REST API, install the `json-server` library using the following command in your terminal. If you already have the `json-server` package installed, skip this step as well.
+
+```bash
+npm install -g json-server
+```
+This will install the JSON server globally on our computer.
+
+Start the React application using the following command.
+```bash
+npm start
+```
+
+### Creating the React ApexCharts component
+In the `src` folder, create a `components` folder. Inside the `components` folder, create a file `charts.js`, that will contain the component.
+
+![folder structure](/engineering-education/data-visualization-in-react-using-apexcharts/folder-structure.png)
+
+Next, create a functional component `MyCharts` as shown below in charts.js.
+#### charts.js
+```javascript
+import React from 'react';
+import Chart from "react-apexcharts";
+
+const MyCharts = () => {
+  const series = [ //Data on the y-axis
+    {
+      name: "Temperature in Celsius",
+      data: []
+    }
+  ];
+  const options = { //data on the x-axis
+    chart: { id: 'bar-chart'},
+    xaxis: {
+      categories: []
+    }
+  };
+
+  return (
+    <div>
+      <Chart
+        options={options}
+        series={series}
+        type="bar"
+        width="450"
+      />
+    </div>
+  )
+}
+
+export default MyCharts;
+```
+On the second line of our file, we are importing Chart from `react-apexcharts`. We then declare two variables, `series` and `options`. We will then set the `series` and `options` props in the `<Chart />` component to these variables.  `<Chart />` component accepts the following props;
+
+- `options`: This is a prop type of object. All the optional chart configurations are passed here.  In the `options` variable, the `chart` key has a value of an object `{ id: 'bar-chart'}`, which is the id of our chart. The key `xaxis` has a value of `categories`, which is the label that will be on the x-axis.
+
+- `series`: This is a prop type of an array. It contains the data that we want to display on the y-axis.
+
+- `type`: This is a prop type of string. It defines the type of chart that we want to display. In our case, a `bar` chart. There are other types such as `line` and `donut`.
+
+- `width`: Can be a string or a number. It defines the width of the chart.
+
+- `height`: Can be a string or a number. It defines the height of the chart.
```

Comment:
You cannot start a sentence with the word `Can`. Same case with with the word `Include` on line 114. "You need to include....."